portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,22 @@
 // limitations under the License.
 
 /**
- * Adds a random fact to the page.
+ * Adds a random greeting to the page.
  */
-function addRandomFact() {
-  const facts =
-      ['I major in computer science.', 'I am learning new langauges.', 'I have traveled to over 30 states.'];
+function addRandomGreeting() {
```

Comment:
I see that addRandomFact() was changed to addRandomGreeting(). But index.html still calls addRandomFact(). I think we can change either the function itself or call addRandomGreeting in index.html. I'll leave it up to you to choose.